datalake-api/src/main/java/com/sequenceiq/sdx/api/model/ModelDescriptions.java

```diff
@@ -195,6 +195,10 @@ public class ModelDescriptions {
 
     public static final String TARGET_MAJOR_VERSION = "The database major version to upgrade to";
 
+    public static final String BACKUP_INCLUDED_DATA = "Data included in the backup (RANGER_PERMISSIONS, HMS_METADATA, RANGER_AUDITS, ATLAS_METADATA)";
```

Comment:
We are considering providing an option for customers to exclude indexes in ATLAS metadata.
Can you break ATLAS_METADATA to ATLAS_METADATA and ATLAS_INDEXES?

Reply:
done